Assets/Scripts/Player/PlayerCombat.cs

```diff
@@ -6,98 +6,119 @@ namespace Player
 {
     public class PlayerCombat : MonoBehaviour
     {
-        [Header("Unity References")] public Animator animator;
-
-        public PlayerInput playerInput;
+        [Header("Unity References")]
+        
+        [SerializeField][Tooltip("The animator object for the player sprite.")]
```

Comment:
Why do we use SerializeField on stuff that's set on `Start()` like the animator?